_posts/en/posts/2016-02-26-zkcp.md

```diff
@@ -0,0 +1,140 @@
+---
+type: posts
+layout: post
+lang: en
+id: en-zkcp-announce
+name: zkcp-announce
+title: The first successful Zero-Knowledge Contingent Payment
+permalink: /en/2016/02/26/zkcp-announce/
+author: gmaxwell
+tags: [privacy]
+version: 1
+excerpt: Announcement for the first Zero-Knowledge Contingent Payment on the Bitcoin Network.
+---
+I am happy to announce the first successful  Zero-Knowledge Contingent Payment (ZKCP) on the Bitcoin Network.
```

Comment:
also, we don't usually capitalize "Bitcoin Network", maybe just "Bitcoin network" or "bitcoin network", but this is a super minor nitpick.